templates/admin.html

```diff
@@ -14,82 +15,122 @@ <h1>Status</h1>
         </form>
     </div>
 
+    <!-- Banana Time -->
     <div class="container">
+        <h1>Banana Time</h1>
         <div class="row">
-            <h1>Time Announcements</h1>
             <div class="row g-3">
                 <form class="row" method="POST">
-                    <input type="hidden" name="formId" value="bananaTime">
-                    <label for="" class="form-label">Banana Time</label>
+                    <input type="hidden" name="form_id" value="set_banana_time">
+                    <h3 for="" class="form-label">Time</h3>
                     <div class="col-auto">
-                        <input class="form-control" type="time" name="bananaTime" value="{{ bananaTime }}">
-                    </div>
-                    <div class="col-auto">
-                        <button class="btn btn-primary">Set</button>
-                    </div>
-                </form>
-            </div>
-
-            <div class="row g-3">
-                <form class="row" method="POST">
-                    <input type="hidden" name="formId" value="morningAnnouncementTime">
-                    <label for="" class="form-label">Morning Announcement Time</label>
-                    <div class="col-auto">
-                        <input class="form-control" type="time" name="morningAnnouncementTime" value="{{ morningAnnouncementTime }}">
+                        <input class="form-control" type="time" name="banana_time" value="{{ announcements['banana_time'].time }}">
```

Comment:
The default `value` here needs to be formatted so that the time is in the format `HH:MM`. Otherwise a user can cause an error by clicking SET on banana time without changing it. As the default time value is in the format HH:MM:SS which the backend currently can't deal with.

Either format this time such that this error cannot occur, or modify the `string_to_time()` function to handle this case. Or both! 😄